tests/rest/admin/test_room.py

```diff
@@ -1437,6 +1438,138 @@ def test_join_private_room_if_owner(self):
         self.assertEqual(private_room_id, channel.json_body["joined_rooms"][0])
 
 
+class MakeRoomAdminTestCase(unittest.HomeserverTestCase):
+    servlets = [
+        synapse.rest.admin.register_servlets,
+        room.register_servlets,
+        login.register_servlets,
+    ]
+
+    def prepare(self, reactor, clock, homeserver):
+        self.admin_user = self.register_user("admin", "pass", admin=True)
+        self.admin_user_tok = self.login("admin", "pass")
+
+        self.creator = self.register_user("creator", "test")
+        self.creator_tok = self.login("creator", "test")
+
+        self.second_user_id = self.register_user("second", "test")
+        self.second_tok = self.login("second", "test")
+
+        self.public_room_id = self.helper.create_room_as(
+            self.creator, tok=self.creator_tok, is_public=True
+        )
+        self.url = "/_synapse/admin/v1/make_room_admin/{}".format(self.public_room_id)
+
+    def test_public_room(self):
+        """Test that getting admin in a public room works.
+        """
+        room_id = self.helper.create_room_as(
+            self.creator, tok=self.creator_tok, is_public=True
+        )
+
+        request, channel = self.make_request(
+            "POST",
+            "/_synapse/admin/v1/make_room_admin/{}".format(room_id),
+            content={},
+            access_token=self.admin_user_tok,
+        )
+        self.render(request)
```

Comment:
I think you need to merge in `develop` since this `render` call is now unnecessary.